ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-nightly.yaml

```diff
@@ -800,33 +786,21 @@ tests:
     dependencies:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
     env:
-      BASE_DOMAIN: qe.devcluster.openshift.com
```

Comment:
@JoelSpeed same above comment. just need to add `FEATURE_SET: TechPreviewNoUpgrade`, no need to delete existing env.